actions.yaml

```diff
@@ -5,7 +5,8 @@ get-primary:
   description: Report primary replica
 
 get-password:
-  description: Fetch the password of the provided internal user of the charm, used for internal charm operations. 
+  description:
```

Comment:
Do we need new line here or it was accidental formatting change?

Reply:
my formatter did this